packages/flutter_tools/test/commands.shard/permeable/analyze_once_test.dart

```diff
@@ -109,7 +118,7 @@ void main() {
         toolExit: true,
       );
```

Comment:
Do you need to revert the change to libMain? Probably under a `finally {}`

Reply:
It gets re-created in `setUp` now